src/utils/fetch-live-event-data.js

```diff
@@ -0,0 +1,28 @@
+const API_KEY = '00411c8fd1b9c3224260259d245dd29890130b94';
```

Comment:
Do we want the api-key hard coded here? Do we have a place where this can be stored?

Reply:
I asked Gregg about this and this key is public, so there is no worry about having it in the source code